### DISCUSSION TITLE
Africable TV

### DISCUSSION THREAD

**Discussion by kone78** [iptv-org/community#648] [Q&A]
Please can you add AFRICABLE TV a Channel from Mali, this is the link:
https://www.lyngsat.com/tvchannels/ml/Africable-TV.html
**Channel Name: AFRICABLE TV
**Channel ID:** AfricableTV.ml
**Country:** MALI
**Language:** FRENCH
**External Link (optional):https://www.lyngsat.com/tvchannels/ml/Africable-TV.html
**Notes (optional):** xxx

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ml.m3u#L2-L3
